cmd/kill/killer/kill_unix.go

```diff
@@ -0,0 +1,29 @@
+// +build linux darwin dragonfly freebsd netbsd openbsd
+
+package killer
+
+import (
+	"syscall"
+)
+
+func Kill(pids []int, safe bool) (map[int]error) {
+	var errm = make(map[int]error)
+	var err error
+
+	for _, pid := range pids {
+		err = syscall.Kill(pid, syscall.SIGTERM)
+		if err != nil {
+			if safe == true {
+				goto errloop
+			}
+			err = syscall.Kill(pid, syscall.SIGKILL)
+			if err != nil {
+				goto errloop
+			}
+		}
+		continue
```

Comment:
weird. A `for` loop with a default continue. Maybe reversing the logic makes it more readable:

```go
	for _, pid := range pids {
		err = syscall.Kill(pid, syscall.SIGTERM)

		if err != nil {
			if safe == true {
				errm[pid] = err
				continue
			}

			err = syscall.Kill(pid, syscall.SIGKILL)

			if err != nil {
				errm[pid] = err
				continue
			}
		}
	}
```

What do you think?